src/notation/view/percussionpanel/percussionpanelmodel.cpp

```diff
@@ -263,6 +288,63 @@ void PercussionPanelModel::setUpConnections()
         case PanelMode::Mode::SOUND_PREVIEW: playPitch(pitch);
         }
     });
+
+    if (!audioSettings()) {
+        return;
+    }
+
+    audioSettings()->trackInputParamsChanged().onReceive(this, [this](InstrumentTrackId partId) {
+        updateSoundTitle(partId);
+    });
+}
+
+void PercussionPanelModel::updateSoundTitle(InstrumentTrackId partId)
```

Comment:
```suggestion
void PercussionPanelModel::updateSoundTitle(const InstrumentTrackId& trackId)
```

(we usually call it "trackId")